modules/olm-adding-pullsecret-catalogsource.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * operators/admin/olm-restricted-networks.adoc
+// * operators/admin/managing-custom-catalogs.adoc
+
+[id="olm-adding-pullsecret-catalogsource_{context}"]
+= Adding pull secrets to catalog sources
+
+If an index image or catalog image is hosted on an authenticated container image registry, also known as a private registry, you can create a pull secret that contains the authentication credentials for the registry. By referencing the secret in a catalog source, the required images can be pulled for use in OperatorHub.
```

Comment:
Probably should mention that if any operator bundle inside the catalog is hosted in an authenticated registry, i.e a secret is needed to access the bundle image, the secret should be passed on via `spec.secrets` for CatalogSource. 
If there are two operators hosted in different private registries, and requires two secrets to access two bundles, both secrets should be passed onto `sepc.Secrets`